packages/legend-application-studio/src/components/editor/side-bar/RegisterService.tsx

```diff
@@ -0,0 +1,139 @@
+ */
+import { useState } from 'react';
+import { observer } from 'mobx-react-lite';
+import { LEGEND_STUDIO_TEST_ID } from '../../LegendStudioTestID.js';
+import { useEditorStore } from '../EditorStoreProvider.js';
+import {
+  ContextMenu,
+  PanelContent,
+  PURE_ServiceIcon,
+  clsx,
+  PlayIcon,
+  Dialog,
+  Modal,
+  TimesIcon,
+  EmptyWindowRestoreIcon,
+  WindowMaximizeIcon,
+} from '@finos/legend-art';
+import type { BulkServiceRegistrationState } from '../../../stores/sidebar-state/BulkServiceRegistrationState.js';
+import { BulkServiceRegistrationEditor } from '../edit-panel/service-editor/BulkServiceRegistrationEditor.js';
+import { noop } from '@finos/legend-shared';
+
+export const RegisterService = observer(
+  (props: { bulkServiceRegistrationState: BulkServiceRegistrationState }) => {
+    const editorStore = useEditorStore();
+    const services = editorStore.graphManagerState.graph.ownServices;
+    const [open, setOpen] = useState(false);
+
+    const [isMaximized, setIsMaximized] = useState(false);
+    const toggleMaximize = (): void => setIsMaximized(!isMaximized);
+
+    const renderTestables = (): React.ReactNode => (
+      <>
+        {services.map((service) => (
+          <ContextMenu key={service._UUID}>
+            <div
+              className={clsx(
+                'tree-view__node__container global-test-runner__explorer__testable-tree__node__container',
+              )}
+            >
+              <div className="global-test-runner__explorer__testable-tree__node__result__icon__type">
+                <PURE_ServiceIcon />
+              </div>
+              <div className="global-test-runner__item__link__content">
+                <span className="global-test-runner__item__link__content__id">
+                  {service.name}
+                </span>
+              </div>
+            </div>
+          </ContextMenu>
+        ))}
+      </>
+    );
+
+    return (
+      <div
+        data-testid={LEGEND_STUDIO_TEST_ID.GLOBAL_TEST_RUNNER}
+        className="panel global-test-runner"
+      >
+        <div className="panel__header side-bar__header">
+          <div className="panel__header__title global-test-runner__header__title">
+            <div className="panel__header__title__content side-bar__header__title__content">
+              REGISTER SERVICES
+            </div>
+          </div>
+          <div className="panel__header__actions side-bar__header__actions"></div>
+          <button
+            className="panel__header__action side-bar__header__action global-test-runner__refresh-btn"
+            onClick={() => setOpen(true)}
+            tabIndex={-1}
+            title="Register All Services"
+          >
+            <PlayIcon />
+          </button>
+          <Dialog onClose={noop} open={open}>
+            <Modal
+              darkMode={true}
+              className={clsx('editor-modal query-builder__dialog', {
+                'query-builder__dialog--expanded': isMaximized,
```

Comment:
lets create separate css classes and not reuse query builder ones

Reply:
created a bulk-service-registration.scss file